Kernel/Coredump.cpp

```diff
@@ -232,6 +232,8 @@ ErrorOr<void> Coredump::write_regions()
         real_region->set_readable(true);
         real_region->remap();
 
+        auto buffer = TRY(KBuffer::try_create_with_size("Coredump Region"sv, region.page_count() * PAGE_SIZE));
```

Comment:
nit: This could probably have a more descriptive name?

Reply:
I tried to make it as descriptive as possible, since I didn't really want to generate strings on-the-fly.